.ci/magic-modules/generate-terraform.sh

```diff
@@ -27,10 +37,12 @@ LAST_COMMIT_AUTHOR="$(git log --pretty="%an <%ae>" -n1 HEAD)"
 bundle install
 
 # Build all terraform products
-bundle exec compiler -a -e terraform -o "${GOPATH}/src/github.com/terraform-providers/terraform-provider-google/"
+# Since not all products have a beta version, generate everything at GA, then
+# generate the ones that have a beta version at beta.
+bundle exec compiler -a -e terraform -o "${GOPATH}/src/github.com/terraform-providers/$PROVIDER_NAME/" $VERSION_FLAG
 
 # Resources that were already using beta APIs before they started being autogenerated
-bundle exec compiler -v beta -p products/compute -t Address,Firewall,ForwardingRule,GlobalAddress,RegionDisk,Subnetwork,VpnTunnel -e terraform -o "${GOPATH}/src/github.com/terraform-providers/terraform-provider-google/"
+bundle exec compiler -v beta -p products/compute -t Address,Firewall,ForwardingRule,GlobalAddress,RegionDisk,Subnetwork,VpnTunnel -e terraform -o "${GOPATH}/src/github.com/terraform-providers/$PROVIDER_NAME/"
 
 # This command can crash - if that happens, the script should not fail.
 set +e
```

Comment:
This is where you'll need to add `--tag terraform-beta` if you want the magician to understand differences in pr template tags.